contributing.md

```diff
@@ -90,6 +90,49 @@ cd lib/<package-name>
 npm run test:coverage
 ```
 
+### VS Code Setup
+
+If you are looking to use VS code as an IDE and want to take advantage of a Test/Debugger Ui, you can install the following Jest package from the marketplace: [Jest](https://marketplace.visualstudio.com/items?itemName=Orta.vscode-jest)
+
+Once installed, you will need to configure the jest environment to properly recognize and debug the tests within the repo. For this, you can place the following config in either the local settings file in the repository 
+
+`.vscode\settings.json` 
+
+or in your VS Code settings in your app data folder 
+
+`%APPDATA%\Roaming\Code\User\settings.json`
+
+```json
+    "jest.autoRun": "off",
```

Comment:
Rather than requiring folks to configure this manually can we add the extension [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/.vscode/extensions.json) and the config [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/.vscode/settings.json) so everyone gets this out of the box?

Reply:
+1 Also, autoRun options seems to be deprecated. New config would be "jest.runMode": "on-demand"